src/setup-transport.coffee

```diff
@@ -0,0 +1,19 @@
+colors = require './colors'
+prefixes = require './prefixes'
+{ getGlobalScope, isBrowser } = require './utils'
+
+setupTransport = (transport) ->
+  globalScope = getGlobalScope()
+  browser = isBrowser(globalScope)
+
+  if browser
+    transport.setLogLevelPrefixes(prefixes.browser)
+    transport.setLogLevelStyles(colors.browser)
```

Comment:
Wouldn't this be better as a direct part of createBrowserTransport/createCliTransport?

Reply:
Yes this makes sense now since it'll be only used in one place.